### DISCUSSION TITLE
support kmbox mouse move please

### DISCUSSION THREAD

**Discussion by xANoyU** [SunOner/yolov8_aimbot#245] [Ideas]
https://github.com/ZCban/kmboxNET

**Comment by SeeMi157** [comment]
这玩意儿不是有手就行吗，我更关心准心和目标一扯一扯的问题，不知道怎么才能让准心一直在人身上

**Reply by zhuqinshu** [reply]
这个怎么移动啊，我使用的kmbox 的move api ，那个移动 和ghub 完全不一样，会缓慢的移动

**Reply by zhuqinshu** [reply]
可以了。，没啥问题了
